# Examen 2

## Situación 1

Un vuelo transatlántico es el vuelo de una aeronave a través del Océano Atlántico, empezando desde el continente americano y terminando en Europa o África.

<img src="map.png" align="middle" width="800">

**1. Si deseamos realizar un viaje transatlántico desde Estados Unidos a Rusia, pensando que el vuelo toma una trayectoria en línea recta, ¿a cuántos kilómetros se encuentran estos dos países? (Considera que el mapa está a una escala de $3:6011\ \rm{km}$.)**

Primero notemos que las coordenadas de los puntos que nos dan son:
* $\rm{A}(17,2)$
* $\rm{B}(6,3)$
* $\rm{EU}(4,7)$
* $\rm{R}(11,8)$

La distancia entre Estados Unidos (EU) y Rusia (R) es

$$ d=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

$$ =\sqrt{[(4)-(11)]^2+[(7)-(8)]^2} $$

$$ =\sqrt{(-7)^2+(-1)^2} $$

$$ =\sqrt{49+1} $$

$$ =\sqrt{50} $$

Entonces EU y R se encuentran  $\sqrt{50}=5\sqrt{2}\approx7.071$ unidades de distancia (sobre el mapa).

Tenemos que hacer regla de tres
\begin{align*}
3\ \rm{unidades} &\ \leftarrow\ 6011\ \rm{km}\\
&\ \nearrow\\
\sqrt{50}\ \rm{unidades}&
\end{align*}

Entonces, la distancia real que tendrá que viajar el avión es

$$\sqrt{50}\cdot6011\div3\ \rm{km}$$

ó

$$\dfrac{6011\sqrt{50}}{3}\ \rm{km}$$

A) $\dfrac{3\sqrt{18}}{6011}$

B) $\dfrac{3\sqrt{50}}{6011}$

C) $\dfrac{6011\sqrt{18}}{3}$

D) $\dfrac{6011\sqrt{50}}{3}$

**Solución:** D)

**2. Si un avión vuela en línea recta desde Estados Unidos a Rusia, ¿cuál es la ecuación general de la recta que modela ese viaje?**

* $\rm{EU}(4,7)$
* $\rm{R}(11,8)$

Primero obtenemos la pendiente para estos puntos:

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ =\dfrac{(8)-(7)}{(11)-(4)} $$

$$ =\dfrac{1}{7} $$

Esperamos que la ecuación de la recta tenga la forma:

$$ y=mx+b $$

Sustituimos el valor calculado de $m$:

$$ y=\frac{1}{7}x+b $$

Despejamos $b$:

$$ b=y-\frac{1}{7}x $$

Sustituimos las coordenadas de alguno de los dos puntos; por ejemplo, $\rm{EU}(4,7)$ :

$$ b=(7)-\frac{1}{7}(4) $$

$$ =7-\frac{4}{7} $$

$$ =\dfrac{49}{7}-\frac{4}{7} $$

$$ =\dfrac{45}{7} $$

Entonces, la ecuación de la recta en su forma ordenada-pendiente es:

$$ y=\dfrac{1}{7}x+\frac{45}{7} $$

Como la queremos en su forma general pasamos todo al mismo lado de la ecuación:

$$ -\dfrac{1}{7}x+y-\frac{45}{7}=0 $$

Vemos que ninguna de las opciones tiene fracciones, excepto la D), pero esa no puede ser porque está en la forma ordenada-pendiente, además de que no coincide con nuestro resultado. Entonces, nos deshacemos del denominador multiplicando toda la ecuación anterior por $7$.

$$ 7\cdot\left(-\dfrac{1}{7}x+y-\frac{45}{7}=0\right) $$

$$ -x+7y-45=0 $$

Lo que acabamos de obtener se parece bastante al inciso C), pero tiene el signo del 45 al revés. Entonces, esa no es la respuesta correcta.

Puede ser que trate de un múltiplo de la ecuación que obtuvimos. Por ejemplo, podemos multiplicar la ecuación anterior por $2$.

$$ -2x+14y-90=0 $$

Como queremos que el $2x$ sea positivo, pasamos todo para el lado derecho de la ecuación.

$$ 0=2x-14y+90 $$

(Lo mismo se puede lograr si se multiplica toda la ecuación por $-1$. **Recuerden:** esto de multiplicar o dividir una ecuación por algo se puede hacer siempre y cuando realizamos dicha operación a ambos lados de la ecuación.)

Como lo que obtuvimos no coincide con la opción A), seguramente se trata de la C). Comprobamos esto:

Podemos multiplicar la ecuación
$$ -x+7y-45=0 $$
por $3$.

$$ -3x+21y-135=0 $$

Como queremos que el $3x$ sea positivo, pasamos todo para el lado derecho de la ecuación.

$$ 0=3x-21y+135=0 $$

A) $2x-14y+6=0$

B) $3x-21y+135=0$

C) $-x+7y+45=0$

D) $y=\dfrac{1}{7}x-\dfrac{45}{7}$

**Solución:** B)

Como extra quisiera decirles que, otra forma de resolver el ejercicio es sustituyendo las coordenadas de los puntos en todas las ecuaciones hasta ver cuál sí se cumple. Por ejemplo:

**Sustitución de $\rm{EU}(4,7)$ en la opción A)**

$$2x-14y+6=2(4)-14(7)+6=8-98+6=-84\neq0$$

**Sustitución de $\rm{R}(11,8)$ en la opción A)**

$$2x-14y+6=2(11)-14(8)+6=22-112+6=-84\neq0$$

Ninguna de las dos ecuaciones se cumplen. Por lo tanto, el inciso correcto **NO** es el A).

**Sustitución de $\rm{EU}(4,7)$ en la opción B)**

$$3x-21y+135=3(4)-21(7)+135=12-147+135=0$$

**Sustitución de $\rm{R}(11,8)$ en la opción B)**

$$3x-21y+135=3(11)-21(8)+135=33-168+135=0$$

Ambas ecuaciones se cumplen. Por lo tanto, el inciso correcto **SÍ** es el B)

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la recta correcta:

<img src="ej2.png" align="middle" width="1000">

**3. ¿Cuál es la ecuación de la recta perpendicular a la recta que modela el viaje entre Estados Unidos y Rusia, y que pasa por Australia?**

Como ya sabemos la pendiente de la recta que pasa por Estados Unidos y Rusia ($m=\dfrac{1}{7}$), la pendiente perpendicular tiene que ser voltear la fracción y cambiar el signo, es decir,

$$ -\dfrac{7}{1} $$

Entonces, la pendiente de nuestra recta es $m^\prime=-7$.

Si esto no queda tan claro pueden pensarlo con la fórmula

$$ mm^\prime=-1 $$

Es esta despejamos la nueva pendiente y sustituimos la pendiente original.

Esperamos que la ecuación de la recta perpendicular a la anterior tenga la forma:

$$ y=m^\prime x+b^\prime $$

Sustituimos el valor calculado de $m^\prime$:

$$ y=-7x+b^\prime $$

Despejamos $b^\prime$:

$$ b^\prime=y+7x $$

Sustituimos las coordenadas de Australia
$$\rm{A}(17,2)$$
ya que sabemos que tiene que pasar por ahí:

$$ b=(2)+7(17) $$

$$ =2+119 $$

$$ =121 $$

Entonces, la ecuación de la recta perpendicular en su forma ordenada-pendiente es:

$$ y=-7x+121 $$

Como la queremos en su forma general pasamos todo al mismo lado de la ecuación:

$$ 7x+y-121=0 $$

A) $7x+y-117=0$

B) $7x-y-117=0$

C) $7x-y-121=0$

D) $7x+y-121=0$

**Solución:** D)

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la recta correcta:

<img src="ej3.png" align="middle" width="1000">

**4. Si el avión vuela en línea recta desde Estados Unidos hasta Rusia y justo aquí gira para ir hacia Australia en línea recta, ¿cuál es el ángulo que tiene girar?**

Lo que queremos es que el vuelo sea de la siguiente forma:

<img src="vuelo_escala.png" align="middle" width="800">

Y lo que nos interesa es encontrar el ángulo que gira el avión al momento de estar sobre Rusia. Por el dibujo esperamos que el ángulo sea mayor que $90^\circ$. 

Podría haber varia maneras de hacer esto. Lo más fácil es obtener ángulo entre rectas. Nos conviene encontrar la pendiente que forma Rusia con Australia:

* $\rm{R}(11,8)$
* $\rm{A}(17,2)$

Primero obtenemos la pendiente para estos puntos:

$$ m=\dfrac{y_2-y_1}{x_2-x_1} $$

$$ =\dfrac{(2)-(8)}{(17)-(11)} $$

$$ =\dfrac{-6}{6} $$

$$ =-1 $$

Como ya sabemos que la pendiente entre Estados Unidos y Rusia es $m=\dfrac{1}{7}$, podemos usar la fórmula de ángulo entre pendientes:

$$ \theta=\tan^{-1}\left(\dfrac{m_2-m_1}{1+m_2m_1}\right) $$

$$ =\tan^{-1}\,\left[\dfrac{(-1)-\left(\dfrac{1}{7}\right)}{1+(-1)\left(\dfrac{1}{7}\right)}\right] $$

In [1]:
((-1)-(1/7))/(1+(-1)(1/7))

-1.3333333333333333

In [2]:
atand(-1.3333333333333333)

-53.13010235415598

$$ \theta\approx-53.13^\circ $$

Recuerden que ignoramos el signo. Como este ángulo es menor que $90^\circ$, queremos el suplementario de éste.

In [3]:
180-53.13010235415598

126.86989764584402

Ahora sí obtuvimos un ángulo que es mayor que $90^\circ$.

A) $143.13^\circ$

B) $90^\circ$

C) $135^\circ$

D) $126.87^\circ$

**Solución:** D)

**5. Al tomar un vuelo que va a varias partes del mundo y pasa por Brasil, un avión viaja en la forma de la ecuación**

$$ \frac{7x^2+5y^2}{5xy+31x+41y-250}=2 $$

**¿Qué tipo de cónica representa la ecuación anterior?**

¿Variables en el denominador? Hay que pasarlas para el otro lado.

$$ 7x^2+5y^2=2(5xy+31x+41y-250) $$

Ahora pasamos todo al mismo lado para tener la ecuación en su forma general:

$$ 7x^2+5y^2=10xy+62x+82y-500 $$

$$ 7x^2-10xy+5y^2-62x-82y+500=0 $$

Comparamos la ecuación que obtuvimos con la ecuación general de segundo grado:

$$ Ax^2+Bxy+Cy^2+Dx+Ey+F=0 $$

Entonces los coeficientes son:

$A=7$, $B=-10$, $C=5$, $D=-62$, $E=-82$, $F=500$

Realmente sólo me importan los primeros tres coeficientes.

Como $B$ no es cero, *invocamos el indicador* (el interior de la raíz cuadrada en *Le Chicharronera*):

$$ I=B^2-4AC $$

Sustituimos

$$ I=(-10)^2-4(7)(5)$$

$$ =100-140$$

$$ =-40$$

Como el indicador es negativo, debe tratarse de una elipse.

A) Elipse 

B) Hipérbola

C) Parábola

D) Recta

**Solución:** A

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la cónica correcta:

<img src="ej4_conica.png" align="middle" width="1000">

Finalmente, les dejo la dirección electrónica de mi gráfica de `Desmos` para que jueguen:

https://www.desmos.com/calculator/ghag3b70az

## Situación 2

Los vértices de un triángulo son $A(-8,7)$, $B(9,-9)$, y $C(0,7)$.

**6. Calcula el perímetro del triángulo.**

Tengo que obtener las longitudes de todos los lados del triángulo:
* $\overline{AB}$
* $\overline{BC}$
* $\overline{AC}$

$$ \overline{AB}=\sqrt{[(9)-(-8)]^2+[(-9)-(7)]^2} $$

$$ =\sqrt{(9+8)^2+(-9-7)^2} $$

$$ =\sqrt{(17)^2+(-16)^2} $$

In [1]:
(17)^2+(-16)^2

545

$$ \overline{AB}=\sqrt{545} $$

$$ \overline{BC}=\sqrt{[(0)-(9)]^2+[(7)-(-9)]^2} $$

$$ =\sqrt{(-9)^2+(7+9)^2} $$

$$ =\sqrt{(-9)^2+(16)^2} $$

In [3]:
(-9)^2+(16)^2

337

$$ \overline{BC}=\sqrt{337} $$

$$ \overline{AC}=\sqrt{[(0)-(-8)]^2+[(7)-(7)]^2} $$

$$ =\sqrt{(8)^2+(0)^2} $$

$$ =\sqrt{8^2} $$

**Recuerden:** Las raíz cuadrada y elevar al cuadrado se cancelan entre sí porque son operaciones contrarias.

$$ \overline{AC}=8 $$

El perímetro de un triángulo es la suma de todos sus lados

$$ P=\overline{AB}+\overline{BC}+\overline{AC}=\sqrt{545}+\sqrt{337}+8 $$

A) $\sqrt{274}+\sqrt{373}+\sqrt{549}$

B) $8+\sqrt{882}$

C) $8+\sqrt{337}+\sqrt{545}$

D) $8\sqrt{183665}$

**Solución:** C)

**7. Encuentra los ángulos internos del triángulo.**

<img src="triangulo_examen.png" align="middle" width="300">

**Forma 1:**

Esperamos que el ángulo:
* $B$ sea agudo y el más pequeño de los tres
* $A$ sea agudo y el mediano de los tres
* $C$ sea obtuso y el más grande de los tres

Entre las opciones sólo hay una respuesta así.

**Solución:** B)

**Forma 2:**

Vamos a resolver esto usando la fórmula de ángulo entre rectas:

$$ \theta=\tan^{-1}\left(\frac{m_2-m_1}{1+m_2m_1}\right) $$

Para esto necesitamos saber la pendiente de cada segmento.

$$ m=\frac{y_2-y_1}{x_2-x_1} $$

* **Pendiente del segmento $\overline{AB}$**

$$ m=\frac{(-9)-(7)}{(9)-(-8)} $$

$$ =\frac{-16}{17} $$

* **Pendiente del segmento $\overline{BC}$**

$$ m=\frac{(7)-(-9)}{(0)-(9)} $$

$$ =\frac{16}{-9} $$

* **Pendiente del segmento $\overline{AC}$**

$$ m=\frac{(7)-(7)}{(0)-(-8)} $$

$$ =\frac{0}{8}=0 $$

* **Ángulo $A$ (es el ángulo que se forma entre los segmentos $\overline{AB}$ y $\overline{AC}$)**

$$ \tan^{-1}\,\left[\frac{\left(-\dfrac{16}{17}\right)-(0)}{1+\left(-\dfrac{16}{17}\right)(0)}\right] $$

$$ =\tan^{-1}\left(-\dfrac{16}{17}\right) $$

In [1]:
atand(-16/17)

-43.264295411071615

Ignoramos el signo negativo.

$$ A\approx43.26^\circ $$

* **Ángulo $B$ (es el ángulo que se forma entre los segmentos $\overline{AB}$ y $\overline{BC}$)**

$$ \tan^{-1}\left(\frac{\left(-\dfrac{16}{17}\right)-\left(-\dfrac{16}{9}\right)}{1+\left(-\dfrac{16}{17}\right)\left(-\dfrac{16}{9}\right)}\right) $$

Se ve súper feo. Vamos haciéndolo por partes:

* numerador

In [2]:
-16/17+16/9

0.8366013071895424

* denominador

In [3]:
1+(-16/17)*(-16/9)

2.673202614379085

* división

In [4]:
0.8366013071895424/2.673202614379085

0.31295843520782396

* lo metemos en la tangente inversa

In [5]:
atand(0.31295843520782396)

17.377951046137117

$$ B\approx17.38^\circ $$

* **Ángulo $C$ (es el ángulo que se forma entre los segmentos $\overline{BC}$ y $\overline{AC}$)**

Como sabemos que la suma de los ángulos internos de cualquier triángulo debe ser $180^\circ$, podemos despejar el ángulo que nos falta:

$$ A+B+C=180^\circ $$

$$ C=180^\circ-A-B $$

In [6]:
180-43.264295411071615-17.377951046137117

119.35775354279129

$$ C\approx119.36^\circ $$

También podemos obtenerlo con la fórmula:

$$ \tan^{-1}\left[\frac{(0)-\left(-\dfrac{16}{9}\right)}{1+(0)\left(-\dfrac{16}{9}\right)}\right] $$

$$ =\tan^{-1}\left(\dfrac{16}{9}\right) $$

In [5]:
atand(16/9)

60.64224645720873

Pero como queremos que el ángulo sea obtuso:

In [7]:
180-60.64224645720873

119.35775354279127

$$ C\approx119.36^\circ $$

**Forma 3:**

Ya conocemos todos los catetos. Como se trata de un triángulo oblicuángulo podemos usar ley de cosenos para encontrar cada ángulo.

<img src="triangulo_examen.png" align="middle" width="300">

Volvamos a usar la notación de que, al cateto opuesto a un ángulo le llamamos con la misma letra que ese ángulo pero minúscula.

* $\overline{AB}=\sqrt{545}=c$
* $\overline{BC}=\sqrt{337}=a$
* $\overline{AC}=8=b$

* **Ángulo $A$**

$$ a^2=b^2+c^2-2bc\,\cos A $$

Despejamos $A$:

$$ A=\cos^{-1}\left(\frac{b^2+c^2-a^2}{2bc}\right)$$

$$ =\cos^{-1}\left[\frac{(8)^2+\left(\sqrt{545}\right)^2-\left(\sqrt{337}\right)^2}{2(8)\left(\sqrt{545}\right)}\right]$$

Lo hacemos por partes:

* numerador

In [8]:
8^2+(√545)^2-(√337)^2

272.0

* denominador

In [9]:
2*(8)*(√545)

373.52376095772007

* división

In [10]:
272.0/373.52376095772007

0.7281999926928029

* lo metemos en el coseno inverso

In [11]:
acosd(0.7281999926928029)

43.26429541107161

$$ A\approx43.26^\circ $$

* **Ángulo $B$**

$$ b^2=a^2+c^2-2ac\,\cos B $$

Despejamos $B$:

$$ B=\cos^{-1}\left(\frac{a^2+c^2-b^2}{2ac}\right)$$

$$ =\cos^{-1}\left[\frac{\left(\sqrt{337}\right)^2+\left(\sqrt{545}\right)^2-(8)^2}{2\left(\sqrt{337}\right)\left(\sqrt{545}\right)}\right]$$

Lo hacemos por partes:

* numerador

In [12]:
(√337)^2+(√545)^2-8^2

818.0

* denominador

In [13]:
2*(√337)*(√545)

857.1230950102791

* división

In [14]:
818.0/857.1230950102791

0.9543553367794739

* lo metemos en el coseno inverso

In [15]:
acosd(0.9543553367794739)

17.377951046137124

$$ B\approx17.38^\circ $$

* **Ángulo $C$**

$$ c^2=a^2+b^2-2ab\,\cos C $$

Despejamos $C$:

$$ C=\cos^{-1}\left(\frac{a^2+b^2-c^2}{2ab}\right)$$

$$ =\cos^{-1}\left[\frac{\left(\sqrt{337}\right)^2+(8)^2-\left(\sqrt{545}\right)^2}{2\left(\sqrt{337}\right)(8)}\right]$$

Lo hacemos por partes:

* numerador

In [16]:
(√337)^2+8^2-(√545)^2

-144.0

* denominador

In [17]:
2*(√337)*(8)

293.7209560109731

* división

In [18]:
-144.0/293.7209560109731

-0.49026123963255896

* lo metemos en el coseno inverso

In [20]:
acosd(-0.49026123963255896)

119.35775354279127

$$ C\approx119.36^\circ $$

**¡Listo!**

A) $A=17.38^\circ$, $B=43.26^\circ$, $C=119.36^\circ$ 

B) $A=43.26^\circ$, $B=17.38^\circ$, $C=119.36^\circ$

C) $A=43.27^\circ$, $B=76.09^\circ$, $C=60.64^\circ$

D) $A=43.27^\circ$, $B=60.64^\circ$, $C=76.09^\circ$

**Solución:** B)

**8. Recordemos que la mediana de un triángulo es la recta que va del vértice al punto medio del cateto opuesto. El punto donde se cruzan las tres medianas de un triángulo se conoce como baricentro. ¿Cuáles son las coordenadas del baricentro de este triángulo?**

Como tenemos que obtener puntos medios usamos las fórmulas.

**Ojo:** sólo es necesario obtener dos puntos medios porque sólo necesitamos encontrar dos rectas para ver dónde se intersectan.

* **Punto medio del segmento $\overline{AB}$**

$$ x=\frac{(-8)+(9)}{2}=\frac{1}{2} $$

$$ y=\frac{(7)+(-9)}{2}=\frac{-2}{2}=-1 $$

Las coordenadas del punto medio del segmento $\overline{AB}$ son $\left(\dfrac{1}{2},-1\right)$.

* **Ecuación de la recta (mediana) que pasa por el vértice $C$ y por el punto medio del segmento de $\overline{AB}$**

$$ m=\frac{(-1)-(7)}{\left(\dfrac{1}{2}\right)-(0)} $$

$$ =\frac{-8}{\frac{1}{2}} $$

$$ =-16 $$

Esperamos que la ecuación sea de la forma:

$$ y=mx+b $$

Sustituimos la pendiente:

$$ y=-16x+b $$

Despejamos $b$:

$$ b=y+16x $$

Sustituimos uno de los dos puntos, por ejemplo $C(0,7)$:

$$ b=(7)+16(0)=7 $$

Entonces la ecuación de la recta tiene la forma

$$ y=-16x+7 $$

* **Punto medio del segmento $\overline{BC}$**

$$ x=\frac{(9)+(0)}{2}=\frac{9}{2} $$

$$ y=\frac{(-9)+(7)}{2}=\frac{-2}{2}=-1 $$

Las coordenadas del punto medio del segmento $\overline{BC}$ son $\left(\dfrac{9}{2},-1\right)$.

* **Ecuación de la recta (mediana) que pasa por el vértice $A$ y por el punto medio del segmento de $\overline{BC}$**

$$ m=\frac{(-1)-(7)}{\left(\dfrac{9}{2}\right)-(-8)} $$

$$ =\frac{-8}{\frac{9}{2}+\frac{16}{2}} $$

$$ =\frac{-8}{\frac{25}{2}} $$

$$ =-\frac{16}{25} $$

Esperamos que la ecuación sea de la forma:

$$ y=mx+b $$

Sustituimos la pendiente:

$$ y=-\frac{16}{25}x+b $$

Despejamos $b$:

$$ b=y+\frac{16}{25}x $$

Sustituimos uno de los dos puntos, por ejemplo $A(-8,7)$:

$$ b=(7)+\frac{16}{25}(-8) $$

$$ =\frac{175}{25}-\frac{128}{25} $$

$$ =\frac{47}{25} $$

Entonces la ecuación de la recta tiene la forma

$$ y=-\frac{16}{25}\ x+\frac{47}{25} $$

En la imagen se ve que todo lo calculamos bien.

<img src="triangulo_medianas.png" align="middle" width="800">

* **Sistema de ecuaciones**

Obtuvimos las ecuaciones

$$ y=-16x+7 $$

$$ y=-\frac{16}{25}\ x+\frac{47}{25} $$

Para la primera ecuación vamos a pasar todas las variables para el mismo lado:

$$ 16x+y=7 $$

Para la segunda ecuación vamos a pasar todas las variables para el mismo lado:

$$ \frac{16}{25}\ x+y=\frac{47}{25} $$

y, además, nos desharemos del $25$ del denominador (es más difícil trabajar con fracciones). Para hacer esto multiplicamos toda la ecuación por $25$:

$$ 16x+25y=47 $$

Entonces tenemos el sistema de ecuaciones

$$
\begin{cases}
16x+y=7\\
16x+25y=47
\end{cases}
$$

Resolvemos mediante la regla de Cramer (determinantes):

\begin{equation}
\Delta=
\begin{pmatrix}
16 & 1\\
16 & 25
\end{pmatrix}
\end{equation}

$$=+(16)(25)-(16)(1)=384$$

\begin{equation}
\Delta_x=
\begin{pmatrix}
7 & 1\\
47 & 25
\end{pmatrix}
\end{equation}

$$=+(7)(25)-(47)(1)=128$$

\begin{equation}
\Delta_y=
\begin{pmatrix}
16 & 7\\
16 & 47
\end{pmatrix}
\end{equation}

$$=+(16)(47)-(16)(7)=640$$

Entonces, las soluciones son:

$$ x=\frac{\Delta_x}{\Delta}=\frac{128}{384}=\frac{1}{3} $$

$$ y=\frac{\Delta_y}{\Delta}=\frac{640}{384}=\frac{5}{3} $$

$(x,y)=\left(\dfrac{1}{3},\dfrac{5}{3}\right)$ son la solución del sistema de ecuaciones, es decir, las coordenadas donde las ecuaciones de las medianas se intersectan.

A) $\left(-\dfrac{1}{3},\dfrac{2}{3}\right)$

B) $\left(\dfrac{1}{3},\dfrac{5}{3}\right)$

C) $\left(-\dfrac{16}{25},-\dfrac{16}{13}\right)$

D) $\left(\dfrac{47}{25},\dfrac{27}{13}\right)$

**Solución:** B)

Finalmente agrego la gŕafica de `Desmos` para que vean que son las coordenadas correctas:

<img src="triangulo_baricentro.png" align="middle" width="1000">

**9. Recordemos que la altura del triángulo es la recta perpendicular a un segmento y que pasa por el vértice opuesto. ¿Cuál es la recta que coincide con la altura que pasa por el vértice $C$?**

Nos piden una recta perpendicular al cateto opuesto a $C$. Este cateto es el segmento $\overline{AB}$, que ya sabemos que tiene pendiente $m=-\dfrac{16}{17}$.

También sabemos que la pendiente perpendicular tiene que ser voltear la fracción y cambiar el signo, es decir,

$$ +\dfrac{17}{16} $$

Entonces, la pendiente de nuestra recta es $m^\prime=\dfrac{17}{16}$.

Si esto no queda tan claro pueden pensarlo con la fórmula

$$ mm^\prime=-1 $$

Es esta despejamos la nueva pendiente y sustituimos la pendiente original.

Esperamos que la ecuación de la recta perpendicular a la anterior tenga la forma:

$$ y=m^\prime x+b^\prime $$

Sustituimos la pendiente:

$$ y=\dfrac{17}{16}x+b^\prime $$

Despejamos $b^\prime$:

$$ b^\prime=y-\dfrac{17}{16}x $$

Sustituimos las coordenadas del vértice $C(0,7)$:

$$ b^\prime=(7)-\dfrac{17}{16}(0)=7 $$

Entonces, la ecuación de la recta perpendicular en su forma ordenada-pendiente es:

$$ y=\dfrac{17}{16}x+7 $$

A) $y=\dfrac{17}{16}x+7$

B) $y=\dfrac{17}{16}x-9$

C) $y=\dfrac{9}{16}x+7$

D) $y=\dfrac{9}{16}x-9$

**Solución:** A)

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente es la recta correcta:

<img src="triangulo_altura.png" align="middle" width="1000">

**10. ¿Cuáles son las coordenadas de la intersección de la recta del ejercicio anterior con la recta que coincide con el segmento $\overline{AB}$?**

Necesitamos encontrar la ecuación de la recta que pasa por el segmento $\overline{AB}$. Ya tenemos la pendiente.

Esperamos que la ecuación de la recta tenga la forma:

$$ y=mx+b $$

Sustituimos la pendiente:

$$ y=-\dfrac{16}{17}x+b^\prime $$

Despejamos $b$:

$$ b=y+\dfrac{16}{17}x $$

Sustituimos las coordenadas de alguno de los dos vértices ($A$ ó $B$), por ejemplo $A(-8,7)$:

$$ b=(7)+\dfrac{16}{17}(-8) $$

$$ =\frac{119}{17}-\dfrac{128}{17} $$

$$ =-\frac{9}{17} $$

Entonces, la ecuación de la recta perpendicular en su forma ordenada-pendiente es:

$$ y=-\frac{16}{17}x-\frac{9}{17} $$

* **Sistema de ecuaciones**

Obtuvimos las ecuaciones

$$ y=\dfrac{17}{16}x+7 $$

$$ y=-\frac{16}{17}x-\frac{9}{17} $$

Para la segunda ecuación vamos a pasar todas las variables para el mismo lado:

$$ -\dfrac{17}{16}x+y=7 $$

y, además, nos desharemos del $16$ del denominador (es más difícil trabajar con fracciones). Para hacer esto multiplicamos toda la ecuación por $16$:

$$ -17x+16y=112 $$

Para la segunda ecuación vamos a pasar todas las variables para el mismo lado:

$$ \frac{16}{17}x+y=-\frac{9}{17} $$

y, además, nos desharemos del $17$ del denominador (es más difícil trabajar con fracciones). Para hacer esto multiplicamos toda la ecuación por $17$:

$$ 16x+17y=-9 $$

Entonces tenemos el sistema de ecuaciones

$$
\begin{cases}
-17x+16y=112\\
16x+17y=-9
\end{cases}
$$

Resolvemos mediante la regla de Cramer (determinantes):

\begin{equation}
\Delta=
\begin{pmatrix}
-17 & 16\\
16 & 17
\end{pmatrix}
\end{equation}

$$=+(-17)(17)-(16)(16)$$

In [21]:
(-17)*(17)-(16)*(16)

-545

$$ \Delta=-545 $$

\begin{equation}
\Delta_x=
\begin{pmatrix}
112 & 16\\
-9 & 17
\end{pmatrix}
\end{equation}

$$=+(112)(17)-(-9)(16)$$

In [22]:
(112)*(17)-(-9)*(16)

2048

$$ \Delta_x=2048 $$

\begin{equation}
\Delta_y=
\begin{pmatrix}
-17 & 112\\
16 & -9
\end{pmatrix}
\end{equation}

$$=+(-17)(-9)-(16)(112)$$

In [23]:
(-17)*(-9)-(16)*(112)

-1639

$$ \Delta_y=-1639 $$

Entonces, las soluciones son:

$$ x=\frac{\Delta_x}{\Delta}=\frac{2048}{-545}=-\frac{2048}{545} $$

$$ y=\frac{\Delta_y}{\Delta}=\frac{-1639}{-545}=\frac{1639}{545} $$

$(x,y)=\left(-\frac{2048}{545},\frac{1639}{545}\right)$ son la solución del sistema de ecuaciones, es decir, las coordenadas donde las ecuaciones de las medianas se intersectan.

A) $\left(-\dfrac{483}{946},\dfrac{1949}{946}\right)$

B) $\left(\dfrac{315}{1988},\dfrac{413}{1988}\right)$

C) $\left(-\dfrac{2048}{545},\dfrac{1639}{545}\right)$

D) $\left(-\dfrac{1748}{1018},-\dfrac{1108}{1018}\right)$

**Solución:** C)

Finalmente agrego la gŕafica de `Desmos` para que vean que efectivamente son las coordenadas correctas:

<img src="triangulo_altura_interseccion.png" align="middle" width="1000">

**11. Calcula el área del triángulo en unidades cuadradas.**

* **Forma 1:**

<img src="area1.png" align="middle" width="400">

Al punto del ejercicio anterior le llamamos $D\left(-\frac{2048}{545},\frac{1639}{545}\right)$.

Recordemos que el área de un triángulo es

$$ A=\frac{\rm{base}\cdot\rm{altura}}{2} $$

Así como está dibujado el triángulo, la base es la longitud del segmento $\overline{AB}$, o sea,

$$ \rm{base}=\sqrt{545} $$

La altura es la longitud del segmento $\overline{CD}$, o sea,

$$ \overline{CD}=\sqrt{(x_1-x_2)^2+(y_1-y_2)^2} $$

$$ =\sqrt{\left[\left(-\frac{2048}{545}\right)-(0)\right]^2+\left[\left(\frac{1639}{545}\right)-(7)\right]^2} $$

In [24]:
√((-2048/545)^2+(1639/545-7)^2)

5.482917592039928

$$ \rm{altura}\approx5.48 $$

Entonces, el área es

In [25]:
(5.482917592039928)*(√545)/2

64.0

$$ A=64\ \rm{unidades}^2 $$

* **Forma 2:**

<img src="area2.png" align="middle" width="400">

Podemos obtener el área del romboide. Esto es lo mismo que el área de un rectángulo:

$$ A=\rm{base}\cdot\rm{altura} $$

Del dibujo vemos que la base es

$$ \rm{base}= 8\ \rm{unidades} $$

Del dibujo vemos que la altura es

$$ \rm{altura}= 16\ \rm{unidades} $$

Entonces el área es

$$ A=(8\ \rm{unidades})\cdot(16\ \rm{unidades})=128\ \rm{unidades}^2 $$

Pero el triángulo es la mitad del romboide. Entonces el área del triángulo es

$$ 64\ \rm{unidades}^2 $$

A) 128

B) $\dfrac{\sqrt{183665}}{2}$

C) $4\sqrt{337}$

D) 64

**Solución:** D)

Finalmente, les dejo la dirección electrónica de mi gráfica de `Desmos` para que jueguen:

https://www.desmos.com/calculator/gkgtweuw7o